index.src.html

```diff
@@ -1252,17 +1014,11 @@ <h4 id="body-extraction" algorithm>
       That is, {{Credential}} objects for `https://example.com` would not be returned directly to
       `https://www.example.com`, but could be offered to the user via the chooser.
 
-  {{PasswordCredential}}s further mitigate the risk of data leakage by never exposing the
-  {{[[password]]}} slot directly to a page's JavaScript, but only allowing its submission to a
-  same-origin server-side endpoint via {{fetch()}}.
 
   ### Same-origin Leakage ### {#passwords-same-origin-leakage}
```

Comment:
This distinction is no longer relevant; I'd simply join them into "Credential Leakage", and talk about general mitigations.